index.js

```diff
@@ -760,23 +766,32 @@ var tick = function() {
 				if(!("units" in node)) {
 					node.units = {};
 				}
-				if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
-					if(node.owner != -1) {
-						nodeWinner = node.owner;
+				if (node.owner == -3) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2675155500/issues/source?bid=3248399&fileBranchId=3307983#l769)